.tool-versions

```diff
@@ -1,2 +1,3 @@
-python 3.11.1
+python 3.11.7
```

Comment:
We have 3.13 to 3.11 at various places, I wonder if this one should be first or latest supported version.. Any take on this?